udata/tests/api/test_dataservices_api.py

```diff
@@ -101,8 +101,29 @@ def test_dataservice_api_create(self):
         self.assertEqual(dataservice.base_api_url, "https://example.org")
         self.assertIsNotNone(dataservice.deleted_at)
 
-        # response = self.get(url_for('api.dataservice', dataservice=dataservice))
-        # self.assert410(response)
```

Comment:
I don't see where you've modified this in this PR?

Reply:
It was already working (only with a new user logged in, so I added it back)

Reply:
Oh my bad, I hadn't see it was commented! Thank you :pray: